tests/bats/api_personal.bats

```diff
@@ -1,70 +1,49 @@
 #!/usr/bin/env bats
 
-# Current build.
-: ${GETH_CMD:=$GOPATH/bin/geth --datadir $BATS_TMPDIR \
-		--lightkdf \
-		--verbosity 0 \
-		--display 0 \
-		--port 33333 \
-		--no-discover \
-		--keystore $GOPATH/src/github.com/ethereumproject/go-ethereum/accounts/testdata/keystore \
-		--unlock "f466859ead1932d743d622cb74fc058882e8648a" \
-	}
-# EF go-ethereum for comparison
-# : ${GETH_CMD:=$GOPATH/bin/mgeth --datadir $GETH_TMP_DATA_DIR --lightkdf --verbosity 0 --port 33333}
-
-setup() {
-	# GETH_TMP_DATA_DIR=`mktemp -d`
-	# mkdir "$BATS_TMPDIR/mainnet"
-  testacc=f466859ead1932d743d622cb74fc058882e8648a
-	tesetacc_pass=foobar
-	regex_signature_success='0x[0-9a-f]{130}'
-}
-
-# teardown() {
-# }
+load api_env
 
 @test "personal_sign1" {
-    run $GETH_CMD \
-				--password=<(echo $tesetacc_pass) \
+    run $GETH_CMD $GETH_OPTS \
+        --password=<(echo $tesetacc_pass) \
         --exec="personal.sign('0xdeadbeef', '"$testacc"', '"$tesetacc_pass"');" console 2> /dev/null
-		echo "$output"
-		[ "$status" -eq 0 ]
+    echo "$output"
+    [ "$status" -eq 0 ]
     [[ "$output" =~ $regex_signature_success ]]
 }
 
 @test "personal_sign2" {
-    run $GETH_CMD \
-				--password=<(echo $tesetacc_pass) \
+    run $GETH_CMD $GETH_OPTS \
```

Comment:
Noting here that the arugment for 'sign' has to be hex. This differed from the docs at one point.